src/clts2vec/parse.py

```diff
@@ -103,23 +109,28 @@ def get_vec(self, sound, vectorize=True):
             >>> print(vec["cont"]
             - 1
         """
-        
-        sound = self.validate(sound)        
-        base_vec = Vector(self.binary_features) #{f: 0 for f in self.binary_features}
+
+        sound = self.validate(sound)
+        base_vec = Vector(self.binary_features)  # {f: 0 for f in self.binary_features}
 
         # check if sound is diphthong or complex
         complex_sound = False
+        sound_to_sound = ""
```

Comment:
Why do you need that here? It is declared if you have a diphthong, if you do not have a diphthong or cluster, the code should not be carried out anyway?

Reply:
True, we do not need that. I think I just added it because the compiler was complaining that `sound_to_sound` might be referenced without being assigned, but that should not be possible.